.github/workflows/release.yml

```diff
@@ -27,11 +27,14 @@ jobs:
           cd ../lfc-scheduler
           go mod tidy
           make build-and-push-image TAG=${{ steps.get_version.outputs.VERSION }}
+          make release-manifests TAG=${{ steps.get_version.outputs.VERSION }}
           cd ../operator
           go mod tidy
           make build-and-push-image TAG=${{ steps.get_version.outputs.VERSION }}
           make controller-gen release-manifests TAG=${{ steps.get_version.outputs.VERSION }}
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
-          files: operator/config/rendered/release.yaml
+          files: |
+            lfc-scheduler/config/rendered/release.yaml
+            operator/config/rendered/release.yaml
```

Comment:
can we combine them also in a single `manifest.yaml`? This way it is easier to install with a single `kubectl apply` :)

Reply:
The two files will now be merged into a single manifest.yaml file